doc/source/index.rst

```diff
@@ -329,6 +329,32 @@ Options:
   - ``min_valid_data_fraction: 10`` - only generate products if at least 10% of covered
   part of scene contains valid data.
 
+Uploading produced data to S3
+*****************************
+
+.. note::
+   Outside of container environments, it is perhaps better to have a separate dispatcher from
+   `Trollmoves <https://github.com/pytroll/trollmoves/>`_ to handle the file transfers. With
+   containers this will require an additional container image, configuration and a use of
+   a shared volume for the temporary local files.
```

Comment:
I'm not sure I agree here. Nothing prevents containers of running multiple processes simultaneously, as indicated here: https://docs.docker.com/config/containers/multi-service_container/
It is of course not recommended, but fixing a dispatcher as a trollflow2 plugin doesn't change the problem in my opinion.
Moreover, I don't understand why there is a doubt that having product generation and dispatching separate is better. 
Aren't we trying to separate concerns in general?

Reply:
For me this is a placeholder until we can write "directly" to S3. With this approach, we don't need to restructure the processing chain later on by removing the additional dispatcher container and adjusting the configuration of the next step in the chain.

I'll reword the text tomorrow.

Reply:
The documentation is reworded and clarified in b13fa0a

Reply:
Used your suggestion.